Desktop/analysis/analysis.cpp

```diff
@@ -92,12 +114,13 @@ void Analysis::clearOptions()
 
 bool Analysis::checkAnalysisEntry()
 {
+
```

Comment:
I was wondering what exactly is the purpose of this method.
This method is called only by setDynamicModule, which is called only by replaceAnalysesOfDynamicModule, so if I understand well, only when an engine has to process a new module? But if I set a breakpoint in this method, the debugger never stops...

Reply:
This is necessary because we have multiple datastructures representing the same data :s

The `ribbonmodel` has its own copy and `Analysis` links to it, so whenever a module is installed it needs to run this to make sure that it still points to the same place. To be able to do this `Analysis::__codedAnalysisEntry` exists.
If you want the debugger to stop here you should install a developer module. Or change the `Description.qml` of an installed developer module.